test-project/Assets/Generated/Source/improbable/gdk/tests/ExhaustiveMapKeyEcsViewManager.cs

```diff
@@ -43,9 +43,10 @@ public void ApplyDiff(ViewDiff diff)
                 }
 
                 var updates = diffStorage.GetUpdates();
+                var dataFromEntity = workerSystem.GetComponentDataFromEntity<Component>();
                 for (int i = 0; i < updates.Count; ++i)
                 {
-                    ApplyUpdate(in updates[i]);
+                    ApplyUpdate(in updates[i], dataFromEntity);
```

Comment:
are these the changes from sami's earlier PR?

Reply:
Seems like it